tests/drum/unit/test_runtime_parameters.py

```diff
@@ -28,15 +28,7 @@ class TestRuntimeParameters:
         "runtime_param_type, payload",
         [
             (RuntimeParameterTypes.STRING, "Some string value"),
-            (
-                RuntimeParameterTypes.CREDENTIAL,
-                {
-                    "credentialType": "s3",
-                    "awsAccessKeyId": "123aaa",
-                    "awsSecretAccessKey": "3425sdd",
-                    "awsSessionToken": "12345abcde",
-                },
-            ),
+            ( RuntimeParameterTypes.CREDENTIAL, { "credentialType": "askdfjsdk" }),
```

Comment:
Is this a valid runtime param?

Reply:
yes-ish.  this is a valid schema, and the code will pass, but it's not actually a valid param.

Reply:
So why the change? s3 also seemed fine-ish? :)

Reply:
1. we have nice s3 examples elsewhere in the same test file
2. a garbage parameter in the test helps show that the code allows arbitrary strings. 

the only thing required for a credential env var is that it has a non-empty credential type.  this test demonstrates the bare minimum and answers the question I had, which was, "What are the actual code requirements for this piece of code".  When I saw the original test and looked at the code, it _seemed_ like anything was ok,  but there was no test confirming that.  then there was also the misleading test that seemed to say the code did much more than it actually could.  I'd like to be able to read the test and quickly determine what the limits are.  an arbitrary value does this better than some expected value that isn't actually checked by the code

Reply:
It was intentionally decided to not to enforce any credential string/type at the drum level for future proof. (cc @elatt )

Reply:
Yes, since drum is sort of disconnected from datarobot, I felt it best to be loosely coupled and not having to chase after DR every time a new credential type was added.

That being said, I don't see any value in tests having garbage data in them. I'm of the belief tests should have values as close the real thing as reasonable. You're testing an implementation detail... we decided (for now) to allow arbitrary strings so when some other team adds a new credential type, it doesn't break drum right away but I feel we should be free to change this.

Reply:
done